easybuild/easyconfigs/m/ml_dtypes/ml_dtypes-0.4.0-gfbf-2023b.eb

```
@@ -0,0 +1,50 @@
+# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2024/02
+easyblock = 'PythonBundle'
+
+name = 'ml_dtypes'
+version = '0.4.0'
+
+homepage = 'https://github.com/jax-ml/ml_dtypes'
+description = """
+ml_dtypes is a stand-alone implementation of several NumPy dtype extensions used
+in machine learning libraries, including:
+bfloat16: an alternative to the standard float16 format
+float8_*: several experimental 8-bit floating point representations including:
+float8_e4m3b11fnuz
+float8_e4m3fn
+float8_e4m3fnuz
+float8_e5m2
+float8_e5m2fnuz
+"""
+
+toolchain = {'name': 'gfbf', 'version': '2023b'}
+
+dependencies = [
+    ('Python', '3.11.5'),
+    ('SciPy-bundle', '2023.11'),
+]
+
+
+use_pip = True
+
+default_easyblock = 'PythonPackage'
```

No need to set the default for PythonBundle

```suggestion
use_pip = True
```